spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -573,115 +611,470 @@ Welcome to the Spring Cloud Data Flow shell. For assistance hit TAB or type "hel
 server-unknown:>
 ----
 ====
-
-. Configure the Data Flow server URI with the following command (use the URL determined in the previous step and the default user name and password):
++
+If the Data Flow Server and shell do not run on the same host, point the shell to the Data Flow server URL.
```

Comment:
We don't need this line. It seems confusing. The next statement can be adjusted to state how to connect Shell to the Server instead.